automated-e2e-test/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
MIT?

Reply:
@skdwriting This license file remained here when I migrated this from another repository I had created. I will remove this License as the qa repository already has the Apache license.

Reply:
Fixed